src/main/java/de/zalando/ep/zalenium/container/DockerContainerClient.java

```diff
@@ -263,36 +264,40 @@ private String generateContainerName(String zaleniumContainerName,
     }
 
     private void loadMountedFolders(String zaleniumContainerName) {
-        if (!this.mntFoldersAndHttpEnvVarsChecked) {
+        if (!this.mntFoldersAndHttpEnvVarsChecked.get()) {
             String containerId = getContainerId(zaleniumContainerName);
-            if (containerId == null) {
+            if (containerId == null)
```

Comment:
Sorry if I am too picky... 
Can you please leave this if with the brackets? All the project uses it. E.g.

```java
if (containerId == null) {
    return;		                  
}
```